cli/src/lib.rs

```diff
@@ -123,7 +123,7 @@ where
             G::from_configuration(
                 args.submit_genesis,
                 RawGenesisBlock::from_path(genesis_path)?,
-                &Some(config.genesis.clone()),
+                Some(&config.genesis.clone()),
```

Comment:
Could it be just:

```rust
Some(&config.genesis)
```

?

Like in the other places

Reply:
Yes